test/extended/util/image/README.md

```diff
@@ -0,0 +1,53 @@
+# Images used by e2e tests
+
+We limit the set of images used by e2e to reduce duplication and to allow us to provide offline mirroring of images for customers and restricted test environments. Every image used in e2e must be part of this utility package or referenced by the upstream `k8s.io/kubernetes/test/utils/image` package.
+
+All images used by e2e are mirrored to:
+
+  quay.io/openshift/community-e2e-images:TAG
+
+for easy mirroring by the `openshift-tests images` command.
+
+## To add a new image:
+
+1. Identify whether your use case can be solved by an existing image:
+   * The standard "shell" image provided by `ShellImage()` that is available on all clusters and has bash + a set of standard tools available. Use this by default.
+   * An upstream image that is provided by `test/utils/image`.
+2. If your use case is novel, you must:
+   * Describe your use case in a PR to this file and have it approved
+   * Define a standard CI build and publish the image to quay.io in the openshift namespace
```

Comment:
@smarterclayton as i understand it, standard CI builds don't publish content to quay.io, you have to add the mirroring config to https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/mapping_origin_4_7 for that to happen.

So is that your expectation?
1) add CI that builds it (makes image available on registry.svc.ci)
2) add mirroring config that publishes the image to openshift namespace on quay (why do we need this?)
3) have automation promote the image to quay mirror location (what location is this?  community-e2e-images?  should this also be done via the mirroring config?)
4) have an OWNER manually push the image to quay.io/openshift/community-e2e-images  (still unclear on the relationship between this and (3), why we need both, in what order to do them, etc)

cc @gabemontero

Reply:
(and is (3) also achieved via config in https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/mapping_origin_4_7 ? )

Reply:
I also don't understand why these instructions tell people to add things to the "init" section of images.go.  That's not what i'd expect if someone adds a new test that ref's a new image, only if they have an explicit exception case.

Finally, the instructions say to contact someone in OWNERS in this dir, but there is no OWNERS file in this dir.